### ISSUE TITLE
Bug report: Crash when creating apk using Taichi - copy lib for arch:x86error | libepic.so: open failed: ENOENT 

### ISSUE THREAD

**Issue by akhilkedia** [taichi-framework/TaiChi#1118]
**Describe the bug**
For some apks, "Create App" with Taichi causes Taichi to crash and the app is not created. 

**To Reproduce**
1. Install Taichi
2. Click Create App (+)
3. Click (+)
4. Select this apk from here - [apk](https://drive.google.com/open?id=19al-2h02jKQ55CDIm9OGwtCjw2PDWbL1)

**Expected behavior**
Taichi install the app and does not crash.

**Smartphone (please complete the following information):**
Magisk version：N/A (non-root)
TaiChi-Magisk module version：N/A
TaiChi version：6.0.3-2321
Which Xposed modules are enabled ?：None
Xposed modules version：N/A
Which Magisk modules are enabled ?：N/A
Magisk modules version：N/A
Device ：Samsung Galaxy S20+
ROM：Stock, Factory Android 10

**Additional context**
1. Output of `adb shell getprop ro.product.cpu.abi` - `arm64-v8a`
2. Output of `adb shell cat /proc/cpuinfo` - `Processor	: AArch64 Processor rev 0 (aarch64)`
3. SoC Model - `Qualcomm Snapdragon 865 (SM8250)`
4. The file libepic.so only seems to be inside the folder `/data/user/0/me.weishu.exp/files/.exp/lib/armeabi-v7a/libepic.so`
3. Here is the relevant debug log - 
```
2020-03-31 19:33:00.447 23820-23863/? I/me.weishu.exp.core.ExposedStripper: do not need to strip.
2020-03-31 19:33:00.447 23820-23863/? I/Creator: lib not exist, copy armeabi-v7a
2020-03-31 19:33:00.448 23820-23863/? I/Creator: native Lib Dir: /data/user/0/me.weishu.exp/cache/apks/com.sds.sso.agent_1585650780278/lib subDirs: [/data/user/0/me.weishu.exp/cache/apks/com.sds.sso.agent_1585650780278/lib/armeabi-v7a, /data/user/0/me.weishu.exp/cache/apks/com.sds.sso.agent_1585650780278/lib/x86]
2020-03-31 19:33:00.448 23820-23863/? I/Creator: copy lib for arch: armeabi-v7a
2020-03-31 19:33:00.467 23820-23863/? I/Creator: copy lib for arch: x86
2020-03-31 19:33:00.467 23820-23863/? W/Creator: copy lib for arch:x86error
    java.io.FileNotFoundException: /data/user/0/me.weishu.exp/files/.exp/lib/x86/libepic.so: open failed: ENOENT (No such file or directory)
        at libcore.io.IoBridge.open(IoBridge.java:496)
        at java.io.FileInputStream.<init>(FileInputStream.java:159)
        at me.weishu.exp.util.IOUtils.copyFile(SourceFile:69)
        at me.weishu.exp.core.Creator.copyNativeLibs(SourceFile:588)
        at me.weishu.exp.core.Creator.nativeCreate(Native Method)
        at me.weishu.exp.core.Creator.ˏ(SourceFile:81)
        at me.weishu.exp.ui.MainActivity.ˉ(SourceFile:971)
        at me.weishu.exp.ui.MainActivity.lambda$qzqc7nV5gxFKloG0_LYQ_gDOwrs(Unknown Source:0)
        at me.weishu.exp.ui.-$$Lambda$MainActivity$qzqc7nV5gxFKloG0_LYQ_gDOwrs.call(Unknown Source:4)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)
     Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
        at libcore.io.Linux.open(Native Method)
        at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
        at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
        at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
        at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7893)
        at libcore.io.IoBridge.open(IoBridge.java:482)
        at java.io.FileInputStream.<init>(FileInputStream.java:159) 
        at me.weishu.exp.util.IOUtils.copyFile(SourceFile:69) 
        at me.weishu.exp.core.Creator.copyNativeLibs(SourceFile:588) 
        at me.weishu.exp.core.Creator.nativeCreate(Native Method) 
        at me.weishu.exp.core.Creator.ˏ(SourceFile:81) 
        at me.weishu.exp.ui.MainActivity.ˉ(SourceFile:971) 
        at me.weishu.exp.ui.MainActivity.lambda$qzqc7nV5gxFKloG0_LYQ_gDOwrs(Unknown Source:0) 
        at me.weishu.exp.ui.-$$Lambda$MainActivity$qzqc7nV5gxFKloG0_LYQ_gDOwrs.call(Unknown Source:4) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) 
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:919) 
```

**Comment by akhilkedia** [comment]
All other apks are created and work perfectly fine with TaiChi on this device. This issue only occurs with this particular apk mentioned above.

**Comment by tiann** [comment]
this is not the root cause, i will check it later.

**Comment by akhilkedia** [comment]
Fixed in latest version (6.4.0)
Closing.
